bin/pycbc_make_skymap

```diff
@@ -542,6 +543,8 @@ if __name__ == '__main__':
     parser.add_argument('--fake-strain-seed', type=int,
                         action=MultiDetOptionAction, metavar="IFO:FAKE_STRAIN_SEED",
                         help="The set of fake-strains-seeds for each detector")
+    parser.add_argument('--rescale-loglikelihood',
+                        help="SNR rescaling factor used in BAYESTAR")
```

Comment:
I'd really describe it as a rescaling of the likelihood, not the SNR. If it was a rescaling of the SNR, it would affect the distance estimates, which it does not.